### DISCUSSION TITLE
MCCGQ01LM won't complete pair but it does successfully interview, tried a second one, worked first time

### DISCUSSION THREAD

**Discussion by 26tajeen** [Koenkk/zigbee2mqtt#17253] [Devices]
I have had a pair of [MCCGQ01LM](https://www.zigbee2mqtt.io/devices/MCCGQ01LM.html#xiaomi-mccgq01lm) sitting in a drawer for a couple of years. I took them out and decided to use them.

I have been struggling to pair the first one for the last half hour. Tried the holding in the button for three secs until you get the three flashes then continuing to 'stimulate' the device every second. No dice. Continually says interviewing, I think it even says successfully interviewed, but then it fails to pair and eventually you get a message saying actually it didn't successfully interview.

Tried the second one from same location, same method, and it paired first time.

Do you sometimes just get 'duds' which won't ever pair?!

**Comment by WizBangCrash** [comment]
Hi @26tajeen,
I noticed you said they have been in your drawer for a couple of years. Have you put new batteries in them before attempting to pair?
I have found my Aqara contact sensors only manage to pair with relatively new batteries. As soon as they are a few months old they will communicate fine, but if I try repairing them they fail until I swap in new batteries.
Even disconnected coin cells will self discharge over time.
